evaluator/debug_logger/plugin.go

```diff
@@ -0,0 +1,77 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package dlogger
+
+import (
+	"context"
+	"encoding/json"
+	"sync"
+
+	"github.com/open-policy-agent/opa/plugins"
+	"github.com/open-policy-agent/opa/plugins/logs"
+	"github.com/open-policy-agent/opa/util"
+)
+
+const PluginName = "debug_decision_logs"
+
+type config struct {
+}
+
+type plugin struct {
+	manager *plugins.Manager
+	mtx     sync.Mutex
+	config  config
+}
+
+func (p *plugin) Start(ctx context.Context) error {
+	p.manager.UpdatePluginStatus(PluginName, &plugins.Status{State: plugins.StateOK})
+	return nil
+}
+
+func (p *plugin) Stop(ctx context.Context) {
+	p.manager.UpdatePluginStatus(PluginName, &plugins.Status{State: plugins.StateNotReady})
+}
+
+func (p *plugin) Reconfigure(ctx context.Context, conf interface{}) {
```

Comment:
This is a no-op function right? Would be better for it to be explicitly no-op (empty function).